src/interactivity/feature.js

```diff
@@ -72,34 +73,52 @@ import FeatureVizProperty from './featureVizProperty';
  * @property {FeatureVizProperty} width
  * @property {FeatureVizProperty} colorStroke
  * @property {FeatureVizProperty} widthStroke
+ * @property {FeatureVizProperty} symbol
+ * @property {FeatureVizProperty} symbolPlacement
  * @property {FeatureVizProperty[]} variables - Declared variables in the viz object
  * @property {function} reset - Reset custom feature vizs by fading out `duration` milliseconds, where `duration` is the first parameter to reset
  * @api
  */
+
 export default class Feature {
     constructor (rawFeature, viz, customizedFeatures, trackFeatureViz, idProperty) {
-        const variables = {};
-        Object.keys(viz.variables).map(varName => {
-            variables[varName] = new FeatureVizProperty(`__cartovl_variable_${varName}`, rawFeature, viz, customizedFeatures, trackFeatureViz, idProperty);
+        this.id = rawFeature[idProperty];
+
+        this._rawFeature = rawFeature;
+        this._featureVizParams = { rawFeature, viz, customizedFeatures, trackFeatureViz, idProperty };
```

Comment:
I think it is better to pass the parameters to `_buildFeatureVizProperty` directly, they are not used anymore after that

Reply:
Fine!

Reply:
I have changed my mind here after the last refactor, as I was using it more in other methods, and I found it convenient to keep it as a private variable